lib/OpenQA/Utils.pm

```diff
@@ -166,6 +168,23 @@ sub testcasedir {
     return $dir;
 }
 
+sub gitrepodir {
+    my %args = (distri => '', version => '', repo => 'tests', @_,);
+    my $path
+      = $args{needles}
+      ? needledir($args{distri}, $args{version})
+      : testcasedir($args{distri}, $args{version});
```

Comment:
Can this be put in a single line or does it get too long for tidy to accept?

Reply:
i think it was done by tidy. not sure.

Reply:
*this change is ready for review* tidy didnt complain